client/src/app/modules/events/participant-list/participant-list.component.ts

```diff
@@ -1,29 +1,98 @@
-import { Component, OnInit, Input, Output } from '@angular/core';
+import { Component, OnInit, Input, Output, EventEmitter, OnChanges } from '@angular/core';
 import { IEventSchema } from '../../../interfaces/event.interface';
+import { IUserSchema } from '../../../interfaces';
 
 @Component({
   selector: 'app-participant-list',
   templateUrl: './participant-list.component.html',
   styleUrls: ['./participant-list.component.scss']
 })
-export class ParticipantListComponent implements OnInit {
+export class ParticipantListComponent implements OnInit, OnChanges {
+
+  private static SEPARATOR_WORD = ';';
+  private static SEPARATOR_LINE = '\n';
+
+  private export_filename: string;
+  private participants: IUserSchema[];
+  private imgExportSrc = '/assets/images/icons/export-light-blue.png';
+  private imgInviteSrc = '/assets/images/icons/invite-light-blue.png';
+  private showInviteDialog = false;
+
   @Input() iEvent: IEventSchema;
-  @Input() participants: any[] = [
-    {name: 'John', surname: 'Doe', gender: 'male'},
-    {name: 'Jane', surname: 'Doe', gender: 'female'},
-    {name: 'Jack', surname: 'Kerouac', gender: 'male'}
-  ];
+  @Output() iEventChange: EventEmitter<IUserSchema[]> = new EventEmitter();
+
+  private static fileHeader = (): string => {
+    return 'Name' + ParticipantListComponent.SEPARATOR_WORD +
+           'Surname' + ParticipantListComponent.SEPARATOR_WORD +
+           'Email' + ParticipantListComponent.SEPARATOR_LINE;
+  }
+  private static fileData = (person: IUserSchema): string => {
+    return person.firstName + ParticipantListComponent.SEPARATOR_WORD +
+           person.lastName + ParticipantListComponent.SEPARATOR_WORD +
+           person.email + ParticipantListComponent.SEPARATOR_LINE;
+  }
 
   constructor() {}
 
-  ngOnInit() {}
+  ngOnInit(): void {
+    // Dummy array for testing until joining events works
```

Comment:
add a [TODO] so we can find it later on, we may forget it & it'll stay in the app